### PR TITLE
[KUNFXTCH-271] Make client-gear great again

### PR DESCRIPTION
**Pull request by catarina-lopes-pt**
Done so far:
- update dependencies and unify package.json for all packages:
  - only have as dependencies the ones needed for that package
  - have the right peer dependencies
  - have the same name for scripts
- fix playground
  - use babel 7
  - upgrade/improve webpack
- add icons route to playground
- remove unused kununu heart icon (not used) in footer and start use plus icon from kununu-icons
- improve lerna usage
  - add `postinstall` to bootstrap the all application
  - add `start` script to start playground from root
  - add `clean` script to delete all node_modules
- update readme
- add back jest dependencies and make tests work again

Later (in different PRs):
- use new kununu linter config (bring back dependencies and webpack hooks) #110 
- add form wrapper to playground #112

### CODE REVIEW

File: packages/kununu-footer/package.json [hunk 1]
```diff
@@ -5,33 +5,25 @@
...
     "@kununu/kununu-logo": "1.1.0",
-    "prop-types": "15.6.2"
+    "@kununu/kununu-theme-v2": "2.0.2",
+    "node-sass-module-importer": "0.1.0",
+    "nukleus": "14.1.8"
   },
   "peerDependencies": {
-    "nukleus": "14.1.2",
-    "react": "16.7.0",
-    "react-dom": "16.7.0"
```
Comment (kaytcat) [inline packages/kununu-footer/package.json]: Shouldn't we have react-dom in peerDeps?

File: packages/kununu-footer/package.json [hunk 2]
```diff
@@ -5,33 +5,25 @@
...
-  "devDependencies": {
-    "@kununu/kununu-theme-v2": "2.0.2",
-    "babel-cli": "6.26.0",
-    "cross-env": "5.2.0",
-    "node-sass": "4.11.0",
-    "node-sass-module-importer": "0.1.0",
-    "nukleus": "14.1.2",
-    "react": "16.7.0",
-    "react-dom": "16.7.0",
-    "rimraf": "2.6.3"
```
Comment (kaytcat) [inline packages/kununu-footer/package.json]: Don't we need this package for the `clean` script below?

File: packages/kununu-footer/package.json [hunk 3]
```diff
@@ -5,33 +5,25 @@
   "main": "dist",
   "author": "kununu",
   "license": "ISC",
-  "devDependencies": {
-    "@kununu/kununu-theme-v2": "2.0.2",
-    "babel-cli": "6.26.0",
-    "cross-env": "5.2.0",
```
Comment (kaytcat) [inline packages/kununu-footer/package.json]: I see this being used below as well 🤔

File: packages/kununu-footer/package.json [hunk 4]
```diff
@@ -5,33 +5,25 @@
   "main": "dist",
   "author": "kununu",
   "license": "ISC",
-  "devDependencies": {
-    "@kununu/kununu-theme-v2": "2.0.2",
-    "babel-cli": "6.26.0",
```
Comment (kaytcat) [inline packages/kununu-footer/package.json]: Do we not need this?

File: packages/kununu-header/.babelrc [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../.babelrc"
```
Comment (kaytcat) [inline packages/kununu-header/.babelrc]: I always thought if you have a .babelrc in the root any sub packages/dirs that use babel will look for it in parent folders?

File: packages/client-gear-playground/app.jsx [hunk 6]
```diff
@@ -1,76 +1,82 @@
...
+// import IconSearch from '../kununu-icons/dist/Search';
+// import IconUser from '../kununu-icons/dist/User';
 
 const at = require('./img/at.gif');
 const ch = require('./img/ch.gif');
 const de = require('./img/de.gif');
 const us = require('./img/us.gif');
 
+import styles from './index.scss';
+import Home from './home';
```
[the remark this answers is not in the document]
Comment (catarina-lopes-pt) [reply packages/client-gear-playground/app.jsx]: I only added as a separate component so I can have different routes for different playgrounds, Icons is already done and I want to add more, so we can live test everything that is not kununu-utils.

File: packages/kununu-header/.babelrc (hunk 5 above)
Comment (catarina-lopes-pt) [reply packages/kununu-header/.babelrc]: It needs the `--root-mode upward` option, now all files are gone :)

File: packages/kununu-footer/package.json (hunk 2 above)
Comment (catarina-lopes-pt) [reply packages/kununu-footer/package.json]: This was moved to the root package.json dependencies, lerna lets you install it there and then run commands that use it.

File: packages/kununu-footer/package.json (hunk 3 above)
Comment (catarina-lopes-pt) [reply packages/kununu-footer/package.json]: This was moved to the root package.json dependencies, lerna lets you install it there and then run commands that use it.

File: packages/kununu-footer/package.json (hunk 4 above)
Comment (catarina-lopes-pt) [reply packages/kununu-footer/package.json]: This was moved to the root package.json dependencies, lerna lets you install it there and then run commands that use it.

File: packages/kununu-utils/kununu-logger/index.js [hunk 7]
```diff
@@ -10,7 +10,9 @@ const minimumLogLevel = process.env.MINIMUM_LOG_LEVEL || 'info';
  * @param  object containing res and req
  * @return string stringified object
  */
-export const formatNodeRequest = ({req, res, label, timeTakenMicros}) => JSON.stringify({
+export const formatNodeRequest = ({
+  req, res, label, timeTakenMicros,
```
Comment (kaytcat) [inline packages/kununu-utils/kununu-logger/index.js]: Don't we normally put these on new lines?

File: packages/kununu-utils/kununu-logger/kununuLogger.spec.js [hunk 8]
```diff
@@ -40,12 +41,15 @@ describe('Returns correct log format with text format', () => {
       };
 
       res.send({
-        formatedRequest: formatNodeRequest({req, res, label, timeTakenMicros: 1000}),
+        formatedRequest: formatNodeRequest({
+          req, res, label, timeTakenMicros: 1000,
```
Comment (kaytcat) [inline packages/kununu-utils/kununu-logger/kununuLogger.spec.js]: here too?

**Pull request merged**